dts/bindings/gpio/intel,sedi-gpio.yaml

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+description: Intel SEDI GPIO
+
+compatible: "intel,sedi-gpio"
+
+include: [gpio-controller.yaml, base.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    peripheral-id:
+      type: int
+      description: Peripheral Instance ID
+      required: true
```

Comment:
Not directly related to this PR, but since all SEDI drivers seem to require this property, would it perhaps make sense to refactor its definition to a common `sedi-device.yaml` file which all SEDI driver bindings include? That way you don't have to define it separately for each binding.

Reply:
good point, @jhedberg I will submit a patch to make that happen later.